### ISSUE TITLE
Can't get beacon/mese crystal shard to create key.

### ISSUE THREAD

**Issue by Alatarius** [pandorabox-io/telemosaic#23]
Not sure what I'm doing wrong. When I right-click the beacon with a mese crystal shard, nothing happens at all. There has been no error message and nothing posted to the debug.txt file either.

**Comment by OgelGames** [comment]
Could you provide a list of all the mods you have installed? I suspect something might be overriding the mese cyrstal shard's right-click function.

**Comment by Alatarius** [comment]
Here is my mod list:
<details>
  <summary>Click to expand</summary>

3d_apple
3d_armor
airtanks
airutils
ambience
amethyst_new
anchor
animalworld
anvil
area_containers
artdeco
auroras
awards
bakedclay
banner
basic_materials
beds
biofuel
biomeinfo
biome_lib
bonemeal
breadcrumbs
bricks_plus
bridger
bucket
bucket_wooden
builtin_item
bweapons_modpack
carpets
castle_farming
castle_gates
castle_lighting
castle_masonry
castle_shields
castle_storage
castle_tapestries
castle_weapons
connected_chests
controls
cottages
cracked_castle
craftguide
darkage
decorations_sea
deepslate
digilines
digtron
display_modpack
doc
doc_basics
doc_encyclopedia
doc_identifier
doc_items
doors
drinks
elevators
ethereal
etherium_stuff
everness
extended_placement
facade
factory
factory_bridges
farming
fireworkz
flow
flowerbeds
flowerpot
formspec_ast
furniture
gadgets_modpack
geodes_lib
glass_stained
handle_schematics
hangglider
hopper
hudbars
illumination
itemframes
item_drop
jumpdrive
leads
leather_armor
livingcaves
livingdesert
livingfloatlands
livingjungle
livingnether
logspikes
loot
lootchest_modpack
lwcomponents
lwscratch
magic_materials
mana
marinara
marinaramobs
mesebox
mesecons
mesecons_window
mesecons_x
meseportals
micupack
minecart
minimap_radar
mobkit
mobs
mobs_animal
mobs_ethereal_bosses
mobs_monster
mobs_npc
mobs_skeletons
mobs_sky
mobs_water
modname_tooltip
modutil
moon_phases
moreblocks
moredecorations
morelights
moremesecons
moreores
moretorches
moretrees
music_modpack
nativevillages
naturalbiomes
nether
new_campfire
npc_talk
npc_talk_edit
obsidianstuff
paintings_gallery
paintings_lib
people
pipeworks
pipunhole
plaster
playereffects
player_monoids
powerbanks
real_world_time
redef
regrowing_fruits
rnd
ropes
scaffolding
servercleaner
shallow_ores
shipwrecks
signs_lib
simplecoins
simple_furniture
skinsdb
slats
stackslabs
stainedglass
stripped_tree
superquest
swamp
technic_plus
technic_recipes
techpack
telemosaic
tga_encoder
toolranks
torch_bomb
tpad
travelnet
ts_furniture
tt
tubelib2
unifieddyes
unified_inventory
unified_stamina
variety
vein_miner
visible_sneak
void_chest
we_undo
wilhelminessounds
wine
woodcutting
working_villages
worldedit
worldedit_hud_helper
worldgate
xcompat
xdecor
xmaps
x_bows
x_enchanting
x_farming
yl_speak_up

</details>

**Comment by OgelGames** [comment]
I think you need to update the `nether` mod, this issue was recently fixed there: https://github.com/minetest-mods/nether/commit/d16b530685d62f6891d8f4d537bf36acf842a077

**Comment by Alatarius** [comment]
Sadly the most recent update was last year and I have the most latest version of nether. I viewed the link but I don't understand it at all. I'm not a coder and any code changes I need an ELI5 explanation.

**Comment by OgelGames** [comment]
Hmm... It seems that the version on ContentDB is not updated. Here's the zip download for the latest version: https://github.com/minetest-mods/nether/archive/refs/heads/master.zip

**Comment by wsor4035** [comment]
they have tag release set up, should make an issue for them to tag a new release or better yet, just change to rolling releases
